projects/step-core/src/lib/services/view-registry.service.ts

```diff
@@ -96,6 +101,12 @@ export class ViewRegistryService implements OnDestroy {
       parentId: 'status-root',
     });
     this.registerMenuEntry('Quota Manager', 'gridquotamanager', 'sidebar', { weight: 50, parentId: 'status-root' });
+    // Sub Menus Bookmarks
+    this.registerMenuEntry('Manage Bookmarks', 'bookmarks', 'edit', { weight: 1, parentId: 'bookmarks-root' });
+    const bookmarks = JSON.parse(this.storage.getItem('BOOKMARKS_BOOKMARKS') || '[]');
```

Comment:
I would prefer to encapsulate the storage layer within the bookmarks service

Reply:
Changed for getStorageBookmarks from bookmarkService